src/api/app/models/repository.rb

```diff
@@ -17,6 +17,7 @@ class Repository < ApplicationRecord
   has_many :architectures, -> { order("position") }, :through => :repository_architectures
 
   scope :not_remote, -> { where(:remote_project_name => nil) }
```

Comment:
If you created the :remote scope... why don't you change this to the new hash syntax too? :smile: 
```ruby
scope :not_remote, -> { where(remote_project_name: nil) }
```